### ISSUE TITLE
Log authors

### ISSUE THREAD

**Issue by altsem** [altsem/gitu#156]
Show commit authors in the log.

May want to make the log format customisable in the future.

**Comment by phush0** [comment]
yeah I am interested in same thing, as I know in L-L in magit you can see authors

**Comment by altsem** [comment]
Followed up on #500 and merged this in 15304fde15c9fcdae1b49a211197cf378599391f
